### ISSUE TITLE
Cant add members to organization or team

### ISSUE THREAD

**Issue by robertorubioguardia** [getmeli/meli#177]
Hi guys.

I tried to invite myself using a different email address and got the invite link but when I open it, it just takes me to the meli auth page. Thinking it could be a caching issue, I tried on an incognito window and even another browser to the same result.

While trying to add a member to a team, the pop-up window says "Add branch" instead of "Add member" and there seems to be no way to a new member from it anyways.

Hope this helps to debug this great app.

Cheers.

**Comment by gempain** [comment]
@robertorubioguardia hi, and thank you for the kind words !

To accept an invite, you have to be logged in, which implies you have to register using one of the available [authentication methods](https://docs.meli.sh/authentication).

This is due to how we model the User <-> Member <-> Organization relationship, and I acknowledge that it's a bit odd because when you receive the invitation, you'd expect to get to the invitation page right away, but right now you first have to log-in, and you then have to click the invitation link again (because we don't save the redirect URL when authenticating). There's room for improvement 😄 

Please let me know if this helps !

**Comment by gempain** [comment]
I have just released v1.0.0-beta.11. The typo in the modal has been fixed 😄

**Comment by robertorubioguardia** [comment]
HI @gempain 

I managed to add GitHub authentication and accept the invite following your instructions and then added the user as a member of a team in my organization, with no admin powers.

Thing is, when I log in from the new invited user, I get a 403 error in the GUI where the teams should be listed.
The API error in the console is:
```
{
"statusCode": 403,
"path": "/api/v1/teams/b6ddb484-0feb-4ae7-bf78-45aeaed6f67f/sites",
"message": "Cannot get team"
}
```
If I add the user to the team with admin powers, there is no error. I have also confirmed the team ID is valid.

Cheers.
